chainweb.cabal

```diff
@@ -602,6 +602,7 @@ executable chainweb-node
         , configuration-tools >= 0.6
         , deepseq >= 1.4
         , directory >= 1.3
+        , filepath >= 1.4
```

Comment:
I am not sure if `</>` is worth the new dependency. We used to be more liberal about dependencies while we were building up the codebase, but have recently transitioning to a mode where we try to reduce the number of dependencies in order to reduce attack surface.

What about adding `</>` to `Chainweb.Utils` or, if we need more path specific functions, adding a`Chainweb.Utils.Filepath` module?

Reply:
`</>` is surprisingly large for cross-platform stuff. `directory` transitively depends on `filepath`, does that factor in at all?

Reply:
If directory already depends on it, let's take the dependency.